src/internal/modules/contact-entry/forms/company-search-select-company.js

```diff
@@ -7,12 +7,13 @@ const getCompanyText = row => {
 };
 
 const getCompanyChoices = companies => {
-  const choices = companies.map(record => {
-    return ({
-      value: record.company.companyNumber,
-      label: getCompanyText(record)
-    });
-  });
+  const choices = [{ value: null, label: `${companies.length} companies found` },
+    ...companies.map(record => {
+      return ({
+        value: record.company.companyNumber,
```

Comment:
I assume the data for this is comping from companies house?  (just thinking that the CRM company `companyNumber` property is null for NALD data)

Reply:
This is correct. This array of options comes from Companies House API.